openstack/nova/templates/api-metadata-deployment.yaml

```diff
@@ -143,7 +143,7 @@ spec:
               readOnly: true
             {{- end }}
             {{- include "utils.proxysql.volume_mount" . | indent 12 }}
-        {{- include "utils.proxysql.container" . | indent 8 }}
+        {{- tuple . .Values.api.config_file.DEFAULT.osapi_compute_workers | include "utils.proxysql.container" | indent 8 }}
```

Comment:
That's the wrong variable. Need to use this: https://github.com/sapcc/helm-charts/blob/aae1048b17c973ec708e44d1ee398dc66cb5ea35/openstack/nova/values.yaml#L225